integreat_cms/cms/views/utils/hix.py

```diff
@@ -0,0 +1,64 @@
+"""
+This file contains functionality to communicate with the textlab api to get the hix-value
+for a given text.
+"""
+from functools import lru_cache
+import json
+import logging
+from urllib.error import HTTPError
+
+from django.conf import settings
+from django.http import JsonResponse
+from django.views.decorators.http import require_POST
+
+from ....textlab_api.textlab_api_client import TextlabClient
+from ....api.decorators import json_response
+
+
+logger = logging.getLogger(__name__)
+
+
+@lru_cache(maxsize=512)
```

Comment:
Technically the size of the buffer is unlimited, because the `get_hix_score` function does not restrict the size of a single request. Do you think that a check for this should be added, or can we trust the users enough to not submit very large texts to the hix request?

Reply:
Hmm, probably this does not cause any problems, but yeah let's at least make sure that the CMS is not DDoSable by sending a few GBs of text :sweat_smile:

Reply:
I added a check now. Also I found out that django by default blocks post-requests with bodies larger than 2.5MB (https://docs.djangoproject.com/en/4.1/ref/settings/#data-upload-max-memory-size) so there would be at least some protection for the worst case.